src/components/icon/icons/bulb.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Bulb({
+  stroke,
+  fill,
+  strokeWidth,
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+  return (
+    <svg style={style} {...rest} viewBox={'0 0 12 20'}>
+      <g stroke="none" strokeWidth={strokeWidth} fill="none" fillRule="evenodd">
+        <g fillRule="nonzero" fill={fill}>
+          <path d="M6.5001,20.0003 L5.5001,20.0003 C5.2241,20.0003 5.0001,19.7763 5.0001,19.5003 C5.0001,19.2243 5.2241,19.0003 5.5001,19.0003 L6.5001,19.0003 C6.7761,19.0003 7.0001,19.2243 7.0001,19.5003 C7.0001,19.7763 6.7761,20.0003 6.5001,20.0003" />
+          <path d="M10.085,9.385 C9.788,9.879 9.382,10.721 9.18,11.261 L8.383,13.388 C8.263,13.708 7.842,14 7.5,14 L4.5,14 C4.158,14 3.737,13.708 3.617,13.388 L2.819,11.261 C2.617,10.721 2.211,9.88 1.914,9.385 C1.905,9.37 1,7.84 1,6 C1,3.243 3.243,1 6,1 C8.757,1 11,3.243 11,6 C11,7.829 10.094,9.371 10.085,9.385 M8,15.5 C8,15.777 7.776,16 7.5,16 L4.5,16 C4.224,16 4,15.777 4,15.5 C4,15.224 4.224,15 4.5,15 L7.5,15 C7.776,15 8,15.224 8,15.5 M7.5,18 L4.5,18 C4.224,18 4,17.777 4,17.5 C4,17.224 4.224,17 4.5,17 L7.5,17 C7.776,17 8,17.224 8,17.5 C8,17.777 7.776,18 7.5,18 M6,0 C2.692,0 0,2.693 0,6 C0,8.114 1.014,9.828 1.057,9.901 C1.323,10.344 1.702,11.128 1.883,11.613 L2.681,13.739 C2.802,14.061 3.034,14.352 3.322,14.573 C3.121,14.828 3,15.151 3,15.5 C3,15.884 3.145,16.235 3.383,16.5 C3.145,16.766 3,17.116 3,17.5 C3,18.327 3.673,19 4.5,19 L7.5,19 C8.327,19 9,18.327 9,17.5 C9,17.116 8.856,16.766 8.617,16.5 C8.856,16.235 9,15.884 9,15.5 C9,15.151 8.879,14.828 8.678,14.573 C8.966,14.352 9.198,14.063 9.319,13.739 L10.116,11.613 C10.298,11.128 10.676,10.344 10.942,9.901 C10.985,9.828 12,8.114 12,6 C12,2.693 9.308,0 6,0" />
+        </g>
+      </g>
+    </svg>
+  );
+}
+
+Bulb.propTypes = {
+  ...Icon.propTypes,
+  stroke: PropTypes.string,
```

Comment:
You only have to put `stroke` here if it is actually usable, which is does not seem to be right now. Please check the other icons for this as well.